redisinsight/ui/src/constants/api.ts

```diff
@@ -138,7 +138,8 @@ enum ApiEndpoints {
   ANALYTICS_SEND_PAGE = 'analytics/send-page',
 
   RDI_INSTANCES = 'rdi',
-  RDI_PIPELINE = 'rdi/pipeline'
+  RDI_PIPELINE = 'pipeline',
```

Comment:
is `pipeline` no longer under `rdi` with this change?

Reply:
pipeline still under rdi. I added new function getRdiUrl.

Reply:
👍